src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="projectTest()">
```

Comment:
Watch out for this since this'll add test code to shared resources.  ie. your teammates would end up pulling this test into their dev environments.